notebooks/utils/notebook_utils.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>please restore outputs for utility notebook, it requires for understanding utils content without reading code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I restored the outputs!